core/src/main/java/io/questdb/cairo/sql/Function.java

```diff
@@ -95,9 +95,11 @@ default void close() {
 
     long getLong(Record rec);
 
-    long getLong128Hi(Record rec);
+    long getLong128Hi(Record rec, long location);
```

Comment:
this is an example why arg names should reflect be realistic and unambiguous. `location` is a memory address, why do we need `rec` as an arg? Why don't we introduce a static method rather than methods like this on various interfaces